### PR TITLE
FIX for the BarcodeMutator and for the properties lang

### PR DESCRIPTION
**Pull request by alinmigea**
barcodes and properties are now loaded according to the market referrer and language selected

@plentymarkets/team-multichannel-koboldmaki

### CODE REVIEW

File: src/Generator/IdealoDE.php [hunk 1]
```diff
@@ -499,9 +499,9 @@ private function buildRow(KeyValue $settings, $variation)
                  */
                 if($checkoutApproved == 'true')
                 {
-                    if($variation['data']['skus']['sku'] != null)
+                    if(!is_null($variation['data']['skus'][0]['sku']) && strlen($variation['data']['skus'][0]['sku']) > 0)
```
Comment (simonamoise) [inline src/Generator/IdealoDE.php]: Same for line 544.

Comment (alinmigea) [inline src/Generator/IdealoDE.php]: @simonamoise modification done :)

**Comment by alinmigea** [comment]
@Desentyer 
the last commit contains the modifications for the multiple market ids, that we wanted to have.
The modifications regarding to the v1.0.13 plugin release, need to be merged just after the [PR of the SkuMutator](https://github.com/plentymarkets/php-pl/pull/2260) is adjusted. Thanks :)

**Comment by Desentyer** [comment]
Was ist mit diesem PR? Ist das noch aktuell?

**Comment by alinmigea** [comment]
This PR was related to the idealo and idealo Direktkauf barcodes, but I could take a look to see if it's needed anymore. I think that some of the modifications are outdated

**Comment by alinmigea** [comment]
The PR is not relevant anymore, the modifications related to the SKUs could be made in a new one.
The PR will be closed.

**Pull request closed without merging**